packit/specfile.py

```diff
@@ -160,6 +160,12 @@ def add_patches(self, patch_list: List[PatchMetadata]) -> None:
         if not patch_list:
             return
 
+        if all([p.present_in_specfile for p in patch_list]):
```

Comment:
I'll keep you annoy until you learn it :-D

```suggestion
        if all(p.present_in_specfile for p in patch_list):
```

Reply:
```
In [3]: %timeit all(bool(i) for i in range(1000))                                     
590 ns ± 6.45 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)

In [4]: %timeit all([bool(i) for i in range(1000)])                                   
73.1 µs ± 601 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```
yeah, yours seems to be 10x faster